app/views/meal_compensations/show.html.haml

```diff
@@ -3,7 +3,7 @@
 -#  or later. See the COPYING file at the top-level directory or at
 -#  https://github.com/puzzle/puzzletime.
 
-- @title ||= "Verpflegungs-Entschädigungen-Positionen"
+- @title ||= "Verpflegungsentschädigung-Positionen"
```

Comment:
```suggestion
- @title ||= "Verpflegungsentschädigungspositionen"
```

Ich bin hierbei aber unsicher, da "Positionen" in PTime ja ein fester, zentraler und damit eigenständiger Begriff ist.

Reply:
Ich glaube das ist schon richtig, weil das zeigt auch effektiv die Buchungspositionen an wo Einträge mit Verpflegungsentschädigung gebucht sind.